src/tracks/SampleTrack.cpp

```diff
@@ -746,10 +746,10 @@ TrackView * SampleTrack::createView( TrackContainerView* tcv )
 
 
 
-TrackContentObject * SampleTrack::createTCO(const MidiTime & pos)
+TrackContentObject * SampleTrack::createTCO(const MidiTime & _pos)
 {
-	SampleTCO * sTco = new SampleTCO(this);
-	sTco->movePosition(pos);
+	SampleTCO * sTco = new SampleTCO( this );
+	sTco->movePosition( _pos );
```

Comment:
I'd prefer not to reintroduce outdated formatting
```suggestion
	SampleTCO * sTco = new SampleTCO(this);
	sTco->movePosition(_pos);
```

I know the rest of the createTCO calls have this spacing, but if you want consistency I'd change those instead, since you're replacing most of the code in them anyway. Otherwise I'd leave them as is.

Reply:
Cool, I'll revert that change and fix the style on the new code. I actually made a mistake about which was the right formatting, since I was so used to adding space on the arguments from other PRs